app/screens/get-started-screen/device-account-modal.tsx

```diff
@@ -18,17 +16,16 @@ import {
   logCreateDeviceAccountFailure,
   logCreatedDeviceAccount,
 } from "@app/utils/analytics"
+import * as Keychain from "react-native-keychain"
```

Comment:
note: I think is also what we should use to store the current session token so that it's not easily extractable.

Reply:
Agreed